cypress/e2e/token-explore.test.ts

```diff
@@ -0,0 +1,74 @@
+import { getTestSelector, getTestSelectorStartsWith } from '../utils'
+
+describe('Testing tokens on uniswap page', () => {
```

Comment:
Name after the file. Don't' use the word "testing", because it's a test file.

eg: `describe('token explore', () => {`